molecule/default/molecule.yml

```diff
@@ -6,9 +6,7 @@ driver:
   name: docker
 lint: |
   set -eux
-  yamllint -c .yamllint.yml .
   ansible-lint -v
-  flake8
```

Comment:
It would be nicer to remove these before converting them to molecule v3 syntax to reduce the amount of changes.